### ISSUE TITLE
[bitnami/cassandra] Error loading initDBConfigMap after upgrade to 5.6.5 version of helm chart

### ISSUE THREAD

**Issue by kudodenko** [bitnami/charts#3739]
**Which chart**:
bitnami/cassandra 5.6.5

**Describe the bug**
Error loading initDBConfigMap after upgrade to 5.6.5 version of helm chart.
Cassandra pod logs:

```
kubectl -n namespace logs cassandra-0
Setting node as password seeder
cassandra 21:44:12.74
cassandra 21:44:12.74 Welcome to the Bitnami cassandra container
cassandra 21:44:12.74 Subscribe to project updates by watching https://github.com/bitnami/bitnami-docker-cassandra
cassandra 21:44:12.75 Submit issues and feature requests at https://github.com/bitnami/bitnami-docker-cassandra/issues
cassandra 21:44:12.75
cassandra 21:44:12.76 INFO  ==> ** Starting Cassandra setup **
cassandra 21:44:12.80 INFO  ==> Validating settings in CASSANDRA_* env vars..
cassandra 21:44:12.88 INFO  ==> Initializing Cassandra database...
cassandra 21:44:13.49 INFO  ==> Deploying Cassandra with persisted data
cassandra 21:44:13.50 INFO  ==> Loading user's custom files from /docker-entrypoint-initdb.d ...
cassandra 21:44:13.51 INFO  ==> Starting Cassandra
cassandra 21:44:13.53 INFO  ==> Checking that it started up correctly
cassandra 21:44:28.60 INFO  ==> Found CQL startup log line
cassandra 21:44:29.78 INFO  ==> Nodetool reported the successful startup of Cassandra
cassandra 21:44:29.79 INFO  ==> Trying to access CQL server @ cassandra-0
cassandra 21:44:30.68 INFO  ==> Accessed CQL server successfully
/opt/bitnami/scripts/libcassandra.sh: line 793: /docker-entrypoint-initdb.d//docker-entrypoint-initdb.d/configmap/..2020_09_21_19_26_10.073209671/cassandra_data.cql: No such file or directory
```

**To Reproduce**
Install or upgrade helm chart of cassandra to 5.6.5 version with --set initDBConfigMap=some-configmap

**Expected behavior**
initDb scripts is uploaded to cassandra

**Version of Helm and Kubernetes**:

- Output of `helm version`:

```
version.BuildInfo{Version:"v3.3.1", GitCommit:"249e5215cde0c3fa72e27eb7a30e8d55c9696144", GitTreeState:"dirty", GoVersion:"go1.15"}
```

- Output of `kubectl version`:

```
Client Version: version.Info{Major:"1", Minor:"19", GitVersion:"v1.19.0", GitCommit:"e19964183377d0ec2052d1f1fa930c4d7575bd50", GitTreeState:"clean", BuildDate:"2020-08-26T21:54:15Z", GoVersion:"go1.15", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"17+", GitVersion:"v1.17.9-eks-4c6976", GitCommit:"4c6976793196d70bc5cd29d56ce5440c9473648e", GitTreeState:"clean", BuildDate:"2020-07-17T18:46:04Z", GoVersion:"go1.13.9", Compiler:"gc", Platform:"linux/amd64"}
```

**Additional context**
Most likely the problem is related to update image from docker.io/bitnami/cassandra:3.11.8-debian-10-r0 to docker.io/bitnami/cassandra:3.11.8-debian-10-r17 in 5.6.5 version.

**Comment by miguelaeh** [comment]
Hi @kudodenko ,
Wich version of the chart were you using previously?

**Comment by kudodenko** [comment]
Hi @miguelaeh,
After rollback to 5.6.4 all works as expected.

**Comment by miguelaeh** [comment]
Hi @kudodenko ,
Thank you for reporting the issue. It seems the commit that introduces the issue was: https://github.com/bitnami/bitnami-docker-cassandra/commit/6148e3bd0d6bf124e7f02f30da1310c033f0082d#diff-24367142aacb086050c5e2796514afdd

It should be fixed with https://github.com/bitnami/bitnami-docker-cassandra/pull/79

**Comment by kudodenko** [comment]
I installed cassandra-5.6.5 again, but this version still has a bug.
Should these changes go into the next version or 5.6.5?
```
kubectl -n cassandra-test logs cassandra-0 -f
Setting node as password seeder
cassandra 11:34:33.72
cassandra 11:34:33.72 Welcome to the Bitnami cassandra container
cassandra 11:34:33.72 Subscribe to project updates by watching https://github.com/bitnami/bitnami-docker-cassandra
cassandra 11:34:33.73 Submit issues and feature requests at https://github.com/bitnami/bitnami-docker-cassandra/issues
cassandra 11:34:33.73
cassandra 11:34:33.73 INFO  ==> ** Starting Cassandra setup **
cassandra 11:34:33.77 INFO  ==> Validating settings in CASSANDRA_* env vars..
cassandra 11:34:33.84 INFO  ==> Initializing Cassandra database...
cassandra 11:34:34.30 INFO  ==> Deploying Cassandra from scratch
cassandra 11:34:34.30 INFO  ==> Starting Cassandra
cassandra 11:34:34.31 INFO  ==> Checking that it started up correctly
cassandra 11:34:54.34 INFO  ==> Found CQL startup log line
cassandra 11:34:55.84 INFO  ==> Nodetool reported the successful startup of Cassandra
cassandra 11:34:55.84 INFO  ==> Password seeder node
cassandra 11:34:55.84 INFO  ==> Trying to access CQL server @ cassandra-0.cassandra-headless.cassandra-test.svc.cluster.local
cassandra 11:34:56.85 INFO  ==> Accessed CQL server successfully
cassandra 11:34:56.85 INFO  ==> Creating super-user root
cassandra 11:34:57.73 INFO  ==> Loading user's custom files from /docker-entrypoint-initdb.d ...
/opt/bitnami/scripts/libcassandra.sh: line 793: /docker-entrypoint-initdb.d//docker-entrypoint-initdb.d/configmap/..2020_09_22_11_34_23.170117795/cassandra_testdata.cql: No such file or directory
```

**Comment by miguelaeh** [comment]
Hi @kudodenko ,
The image that has the fix is `3.11.8-debian-10-r19 release`, the next release of the chart should have the image updated to this tag. I am forcing a new release right now.
Regards.

**Comment by kudodenko** [comment]
Thanks a lot, @miguelaeh!
